bindings/matlab/deps.bat

```diff
@@ -0,0 +1,33 @@
+set root=%cd%
+
+// libiio
+mkdir -p deps
+wget https://github.com/analogdevicesinc/libiio/releases/download/v0.23/Windows-VS-16-2019-x64.zip -O libiio-win.zip
```

Comment:
I think we can point this to v0.24 since it won't change for the next libm2k release